dom.bs

```diff
@@ -6354,6 +6355,10 @@ given a <var>document</var>, <var>localName</var>, <var>namespace</var>, and opt
 <a>attribute</a> <var>attribute</var> to <var>value</var>, run these steps:
 
 <ol>
+<li><p>Set <var>value</var> to the result of calling <a>Get Trusted Types-compliant attribute
```

Comment:
Nit: indentation is wrong. And DOM doesn't do newlines in phrasing-level elements (i.e., wrap before `<a>`).

Reply:
Question: this ordering means that if _value_ is changed by TT the mutation record reflects that. Does that have test coverage?

Reply:
Fixed the formatting, thanks for pointers. You're correct re: MutationObserver.

@mbrodesser-Igalia, we need to add a MutationObserver test (https://jsfiddle.net/014ze36t/2/) to WPT.